app/HutchAgent/Services/WorkflowTriggerService.cs

```diff
@@ -0,0 +1,69 @@
+using System.Diagnostics;
+using System.Text;
+using HutchAgent.Models;
+using Microsoft.Extensions.Configuration;
+
+namespace HutchAgent.Services;
+
+public class WorkflowTriggerService
+{
+  private readonly IConfiguration _configuration;
+
+  public WorkflowTriggerService(IConfiguration configuration)
+  {
+    _configuration = configuration;
+  }
+
+  /// <summary>
+  /// Install and run WfExS given 
+  /// </summary>
+  /// <exception cref="Exception"></exception>
+  public void TriggerWfexs()
+  {
+    const string cmd = "bash";
+    string activateVenv = "source " + _configuration["VirtualEnvironmentPath"];
```

Comment:
can we actually use the nice options model class, instead of string key lookups on IConfiguration?

You'll need to configure DI (i.e. using the Generic Host)

Reply:
Yeah can do. I wasn't sure whether to set up the host for DI or if we wanted to keep it a plain console app so I went with the simple option first.